admin_manual/configuration/user/user_auth_ldap.rst

```diff
@@ -396,6 +396,13 @@ Nested Groups:
   By default a group will only contain users, so enabling this option isn't necessary.
   However, if groups are contained inside groups, and this option is not enabled, any groups contained within other groups will be ignored and not returned in search results.
 
+  In Active Directory you have to enter this specific filter in order for the ownCloud LDAP app to be able to find users in nested groups.
```

Comment:
Where in AD do you have to enter this specific filter ?

Reply:
Yeah, I have to rewrite that. You have to enter this filter in ownCloud, not in AD